### PR TITLE
[Docker] Add needed tools to run perf

### PR DESCRIPTION
**Pull request by sitalkedia**
Installing needed tools to run utilities like `perf` on the docker container.

### CODE REVIEW

**Review by rustielin — APPROVED** [review]
Nice LGTM. Looking forward to see the runbook

File: docker/validator/Dockerfile [hunk 1]
```diff
@@ -26,6 +26,9 @@ FROM debian-base AS prod
 
 RUN apt-get update && apt-get install -y libssl1.1 && apt-get clean && rm -r /var/lib/apt/lists/*
 
+### Needed to run debugging tools like perf
+RUN apt-get update && apt-get install -y linux-tools-4.19 sudo procps
```
Comment (msmouse) [inline docker/validator/Dockerfile:30]: n00b q: why do we need to `apt-get update` repeatedly?

Comment (sitalkedia) [reply docker/validator/Dockerfile:30]: Actually we might not need it - but this seems to be the pattern here :D

**Pull request closed without merging**